### PR TITLE
Songs for each venue are saved on Firebase

### PR DESCRIPTION
**Pull request by pfernandez01**

### CODE REVIEW

File: www/app/musicsearch/service.js [hunk 1]
```diff
@@ -5,11 +5,17 @@
...
   /* @ngInject */
-  function SearchService($http, env) {
+  function SearchService($http, env, $stateParams, $firebaseArray) {
     this.searchYoutubeAPI = searchYoutubeAPI;
+    this.pushVideoIDToVenue = pushVideoIDToVenue;
+
+    var ref = new Firebase(env.firebaseApiUrl);
+    var videoIDPath = ref.child($stateParams.venuePath + '/queue');
+    var songs = $firebaseArray(videoIDPath);
+    //console.log(videoIDPath);
```
Comment (emmurillo) [inline www/app/musicsearch/service.js]: Remove it

File: www/app/musicsearch/service.js [hunk 2]
```diff
@@ -23,5 +29,12 @@
       };
       return $http.get(env.youtubeAPIURL, { params: params });
     }
+
+    function pushVideoIDToVenue(youTubeVideoInfo) {
+      return songs.$add(youTubeVideoInfo);
+      //var pathReadyToPush = ref.child(videoIDPath);
+      //return pathReadyToPush.$add();
+      //console.log(videoIDPath);
```
Comment (emmurillo) [inline www/app/musicsearch/service.js]: Remove comments

File: www/app/qrscanner/controller.js [hunk 3]
```diff
@@ -22,13 +24,14 @@
       }
 
     function handleScanSuccess (qrcodeData) {
-      var qrcodeDataID = qrcodeData.text;
+      venueFirebasePath = qrcodeData.text;
       if (!qrcodeData.cancelled) {
         retrieveDataFromLocalID(qrcodeDataID);
```
Comment (emmurillo) [inline www/app/qrscanner/controller.js]: This will crash as `qrcodeDataID` no longer exists, be careful!

File: www/app/musicsearch/controller.js [hunk 4]
```diff
@@ -29,5 +31,21 @@
         template: error.data.error.message
       });
     }
+
+    function addSongToQueue (videoParameters) {
+      vm.videoInfo.videoID = videoParameters.id.videoId;
```
Comment (emmurillo) [inline www/app/musicsearch/controller.js]: video info could be a local variable as is only used in this method

File: www/app/musicsearch/controller.js [hunk 5]
```diff
@@ -12,6 +12,8 @@
     var vm = this;
     vm.videos = {};
     vm.search = search;
+    vm.addSongToQueue = addSongToQueue;
+
```
Comment (joserv2812) [inline www/app/musicsearch/controller.js:16]: delete Line change

**Pull request merged**